tests/test_algo_common.py

```diff
@@ -113,7 +113,7 @@ def __str__(self):
 magnitude_sparsity_params = {'schedule': 'multistep',
                              'multistep_steps': [1, 2],
                              'multistep_sparsity_levels': [0, 0.3, 0.5]}
-filter_pruning_params = {'schedule': 'exponential', 'num_init_steps': 0, 'pruning_steps': 2}
+filter_pruning_params = {'schedule': 'exponential', 'num_init_steps': 0, 'pruning_steps': 3}
```

Comment:
This test was updated because I changed the pruning schedule. From documentation
> `pruning_steps` - the number of epochs during which the pruning rate target is increased from `pruning_init` to `pruning_target` value.

i.e. on epoch with index `pruning_steps - 1` we should set `pruning_target` value.
It was not true before my PR. I fixed it.